.changes/next-release/Feature-f26c3991-fc62-4478-b9bd-7c92d272b009.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Added support for overriding conflicting files during event schemas download code bindings operation"
```

Comment:
```suggestion
    "description": "If a file conflict is detected when downloading event schemas code bindings, a confirmation prompt is shown"
```

Reply:
Sure, thanks for suggestion. Will update.